config/esw.yaml

```diff
@@ -218,13 +234,13 @@ science:
   # These position are random for now
   syringe_servo_positions:  # This is in degrees
     site_A:
-      pushed: 80
+      pushed: 70
       start: 100
     site_B:
       pushed: 60
       start: 90
     site_C:
-      pushed: 10
+      pushed: 0
```

Comment:
Just confirming, this one is supposed to be 50 degrees difference instead of 30?